### PR TITLE
Organization API / createRepo()

### PR DESCRIPTION
**Pull request by marcosomma**
Sorry, I accidentally close the [previous PR](https://github.com/michael/github/pull/293)

### CODE REVIEW

**Comment by marcosomma** [comment]
hi there, any news about this PR?

File: test/test.org.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+         password: testUser.PASSWORD,
+         auth: 'basic'
+      });
+      organization = github.getOrg();
+   });
+
+   it('should create an organisation repo', function(done) {
+      var repoTest = Math.floor(Math.random() * (100000 - 0)) + 0;
+      var options ={
+          "orgname": testUser.ORGANIZATION,
```
Comment (AurelioDeRosa) [inline test/test.org.js]: Properties of an object, unless they are an invalid identifier, don't need to be in quotes. Can you remove them?

File: test/test.org.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+         username: testUser.USERNAME,
+         password: testUser.PASSWORD,
+         auth: 'basic'
+      });
+      organization = github.getOrg();
+   });
+
+   it('should create an organisation repo', function(done) {
+      var repoTest = Math.floor(Math.random() * (100000 - 0)) + 0;
+      var options ={
```
Comment (AurelioDeRosa) [inline test/test.org.js]: There is a space missing between the `=` and the `{`. Have you run `npm run lint`?

**Comment by marcosomma** [comment]
done :)!

**Comment by marcosomma** [comment]
hi there, any news about this PR?

**Comment by AurelioDeRosa** [comment]
Merged. Thank you.

**Comment by clayreimann** [comment]
@marcosomma Could you add me to the [github-api-tests](https://github.com/github-api-tests) org so we can extend our support (and testing) of organizations?

**Comment by marcosomma** [comment]
@clayreimann Sorry for answer that late `-_-'` but actually I used the credentials stored in this file `test/user.json` to admin the [guithub-api-tests](https://github.com/github-api-tests) organization.

**Pull request closed without merging**